about.html

```diff
@@ -0,0 +1,165 @@
+        <div></div>
+        <div></div>
+      </div>
+
+      <!-- Desktop Navigation -->
+      <nav class="desktop-nav-1">
+        <ul class="flex">
+          <li>
+            <a href="$"><i class="fa-brands fa-facebook-f"></i></a>
+          </li>
+          <li>
+            <a href="$"><i class="fa-brands fa-twitter"></i></a>
+          </li>
+          <li><a href="#">English</a></li>
+          <li><a href="#">My Page</a></li>
+        </ul>
+      </nav>
+
+      <nav class="desktop-nav-2 flex">
+        <a class="logoimg" href="./index.html"
+          ><img src="image/glass_red_icon1.png" alt="Logo"
+        /></a>
+        <ul class="flex">
+          <li><a href="./index.html">Home</a></li>
+          <li><a href="#">Menu</a></li>
+          <li><a href="#">Gallery</a></li>
+          <li><a href="#">Order</a></li>
+          <li><a href="#">News</a></li>
+          <li><button type="button">Contact Us</button></li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <section class="about-intro flex">
+        <div class="intro-heading flex about-intro-heading">
+          <h2>Welcome! Food Lovers!</h2>
+          <h1>
+            Rejovan's Kitchen <br />
+            Serves Delicious <br />
+            Meal
+          </h1>
+          <hr />
+        </div>
+        <p class="mobile-paragraph about-mobile-paragraph">
+          Here in rejovan's kitchen we take our customers very important, we
+          have well expertise chefs and our serves are very professional.
+        </p>
+        <p class="about-Email-paragraph">
+          Please contact us via Email for any further questions or you want to
+          place Order
+        </p>
+        <a
+          class="about-Email-paragraph about-Email"
+          href="rejoice:contactus@rejovankitchen.org"
+          >contactus@rejovankitchen.org</a
+        >
+      </section>
+    </main>
+
+    <section class="OurLogo flex">
+      <h2>Rejovan's Kitchen Logo</h2>
+      <hr />
+      <p class="about-Email-paragraph">
+        The Logo is generated from iconfinder site.
+      </p>
+      <div class="aboutOurLogo flex">
+        <img src="image/glass_red_icon1.png" alt="rejovan's kitchen Logo" />
+      </div>
+    </section>
+
+    <section class="OurLogo flex">
+      <h2>some of our popular dish</h2>
+      <hr />
+      <p class="about-Email-paragraph">take a look at our popular dishes.</p>
+      <div class="desktop-dishes flex">
+        <div class="dishes">
+          <div class="dishes-overlay"></div>
+          <section class="dishes-text">
+            <h2>recent</h2>
+            <span class="flex">Delicious beef chicken sauce</span>
+          </section>
+        </div>
+
+        <div class="dish">
+          <div class="dish-overlay"></div>
+          <section class="dish-text">
+            <h2>latest</h2>
+            <span class="flex">fried vegetable sauce</span>
+          </section>
+        </div>
+      </div>
+    </section>
```

Comment:
- [ ] Kindly fix the images and make the layout like the original in the mobile view.

![image](https://user-images.githubusercontent.com/38635158/175650152-0c7a211a-8c75-4967-a318-d24715327ceb.png)
__
### Original
![image](https://user-images.githubusercontent.com/38635158/175650186-9199cf6f-5b76-4ddc-83fe-2f3a0f615d29.png)